modules/rodoo-about.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/run_once_duration_override/index.adoc
+
+:_content-type: CONCEPT
+[id="run-once-about_{context}"]
+= About the {run-once-operator}
+
+{product-title} relies on run-once pods to perform tasks such as deploying a pod or performing a build. Run-once pods are pods that have a `RestartPolicy` of `Never` or `OnFailure`.
+
+Cluster administrators can use the {run-once-operator} to force a limit on the time that those run-once pods can be active. Once the time limit expires, the cluster will try to actively terminate those pods. The main reason to have such a limit is to prevent tasks such as builds to run for an excessive amount of time.
+
+To apply the run-once duration override from the {run-once-operator} to run-once pods, you must enable it on each applicable namespace.
+
+// TODO: Mention which takes precedence if the override is set but the pod had a manual value?
```

Comment:
@ingvagabund What happens if the pod already had a manual activeDeadlineSeconds value set? Do we need to mention anything about conflicts/which takes precedence?

Reply:
If the value is already set but is higher than what is enforced by the admission, it is overridden as well. I.e. the minimum of both is picked: https://github.com/openshift/run-once-duration-override/blob/main/pkg/runoncedurationoverride/mutator.go#L27